docker/README.md

```diff
@@ -0,0 +1,36 @@
+This docker image helps you test out your changes quickly.
+
+### First clone the repo
+```sh
+$ git clone git@github.com:sendgrid/sendgrid-nodejs.git
+```
+
+### Inside the repo create an example file
+```sh
+$ touch example.js
```

Comment:
Would you be up for adding a simple example.js in the README? This way someone can copy/pasta and then test - this adds an experiential positivity that boosts confidence.